### ISSUE TITLE
Keyhandler

### ISSUE THREAD

**Issue by TomasLongo** [codex-team/editor.js#746]
Hi,

is there a way to add custom keyhandlers?

My case: I want to listen to `ctrl + enter` in order to save my doc.

Thanks in advance,
Tomas

**Comment by khaydarov** [comment]
@TomasLongo Hi!
Editor.js provides API to save the contents outside the instance

```js
var editor = new EditorJS();
editor.save(); // here editor saves your data and returns JSON object
``` 

You can wrap method `save()`, like

```js
document.addEventListener('keydown', function() {
  // here you need to check CTRL+ENTER press before firing `save` method
  editor.save().then(...);
}, false);
```

OR another easier way is to use some library for shortcuts
We provide our own: https://github.com/codex-team/codex.shortcuts

**Comment by TomasLongo** [comment]
@khaydarov Thanks for getting back.

The actual saving is not the problem. Im not able to capture the `ENTER` key (`CTRL` works, though). 
It seems like `ENTER` is intercepted by the editor. 

Am I missing something?

Tomas

**Comment by khaydarov** [comment]
@TomasLongo 
Yes, ENTER keypress is intercepted by the Editor, but only inside the Block or when Block is focused. The problem is that we prevent default behaviour and stop propagation that's why you are not able to capture ENTER. 
We're working on this problem

**Comment by TomasLongo** [comment]
Thanks for replying. Glad to here that you're working on this.

Does an issue exist on that topic, which I could subdscribe to?

**Comment by romainhe** [comment]
Any news or workaround to capture `enter` key event ?

**Comment by bencohendev** [comment]
I'm also interested in whether any progress has been made here. I need to be able to intercept and stop `enter`

**Comment by jeasoncc** [comment]
hey, any news ?

**Comment by calumk** [comment]
i think This is required to help fix some issues in my columns plugin [@calumk/editorjs-columns](https://github.com/calumk/editorjs-columns)
